web/src/clients/chat-assistant.ts

```diff
@@ -0,0 +1,64 @@
+type ReceivedMessage = AssistantResponseMessage
```

Comment:
Rename file and `interface`s etc to reflect "model chat" rather than "chat assistant"